Add note on include anon users flag (#5542)
segmentio/segment-docs: src/engage/audiences/index.md

![Creating an Engage Audience from the conditions list](/docs/engage/images/audience_condition_list.png)

> info ""

> > The `Include anonymous users` checkbox determines which external IDs need to exist on a profile for the user to be included in the audience:

Reviewer: ```suggestion
> The **Include Anonymous Users** checkbox determines which external IDs need to exist on a profile for Segment to include the user in the audience:
```

> > **Include anonymous users unchecked**: `user_id` or `email` or `android.idfa` or `ios.idfa`

Reviewer: ```suggestion
> - **Include Anonymous Users not selected**: `user_id`, `email`, `android.idfa`, or `ios.idfa`
```

> > **Include anonymous users checked**: `user_id` or `email` or `android.idfa` or `ios.idfa` or `anonymous_id`

Reviewer: ```suggestion
> - **Include Anonymous Users selected**: `user_id`, `email`, `android.idfa`, `ios.idfa`, or `anonymous_id`
```
